docs/TuxxAnimations.md

```diff
@@ -111,93 +118,138 @@ A numeric id can be specified, in which case all elements will receive that key.
+The duration property, as the name would imply, determines the length of time for the animation sequence. Duration accepts a string with any standard CSS suffix or number input, but **beware** that numbers `will resolve in milliseconds`. The following two examples will both perform a 2 second long Fade animation:
 
 ```javascript
-  <Fly duration="2s">
+  <Fade duration="2s">
     <h1>Hello World</h1>
-  </Fly>  //or
-  <Fly duration={2000}>
+  </Fade>  //or
+  <Fade duration={2000}>
     <h1>Hello World</h1>
-  </Fly>
+  </Fade>
 ```
 
 ##### <a id="animationProps-delay"></a>Delay Property - type: STRING or NUMBER - optional [#](#animationProps-delay)
 The delay property sets a delay on the animation before it activates. The delay property is similar to the duration property in that it accepts either a string or number input. The following example will add a 1 second delay to our animation:
 
 ```javascript
-  <Fly duration={2000} delay={1000}>
+  <Fade duration={2000} delay={1000}>
     <h1>Hello World</h1>
-  </Fly>
+  </Fade>
 ```
 
 ##### <a id="animationProps-easing"></a>Easing Property - type: STRING - optional [#](#animationProps-easing)
 The easing property determines what motion path the animation takes when presented. Easing only accepts a string which can take any custom cubic bezier animation path you wish. For example:
 
 ```javascript
-  <Fly duration={3000} delay={1000} easing="cubic-bezier(0.680, -0.550, 0.265, 1.550)">
+  <Fade duration={3000} delay={1000} easing="cubic-bezier(0.680, -0.550, 0.265, 1.550)">
     <h1>Hello World</h1>
-  </Fly>
+  </Fade>
 ```
 
 The Easing property also accepts any of the Tuxx default easings as a string:
 
 ```javascript
-  <Fly duration={3000} delay={1000} easing="easeInOutBack">
+  <Fade duration={3000} delay={1000} easing="easeInOutBack">
     <h1>Hello World</h1>
-  </Fly>
+  </Fade>
 ```
 
 The Tuxx default easings and their motion paths are illustrated below. The 'Linear' path is not shown, but it is purely a straight line motion path.
 
 ![Easings](http://i.imgur.com/2XAGQgq.png "Easings")
 
+##### <a id="animationProps-vector"></a>Vector Property - type: OBJECT - optional - requires `Origin` & `Magnitude` [#](#animationProps-vector)
+The vector property allows you set where your animation will orignate from and the magnitude of that entrance and exit to that origin. The vector property ccepts an object and expects two keys within that object: `origin` and `magnitude`. Magnitude accepts a number value and Origin accepts any single item of or any combination of two items from the the following list of acceptable origins:  'top', 'bottom', 'left' and 'right'. For example, if you wanted your elements to originate from the bottom right corner of the containing animation component with a magnitude of 50 pixels, it's as easy as:
```

Comment:
In third sentence recommend changing Magnitude to 'magnitude' and Origin to 'origin'.